GameCode.py

```diff
@@ -352,7 +359,11 @@ def start():
             print("A country's tension goes up if a war is lost.")
             print("A country's tension goes down if a war is won.")
             print("If a country's tension is above 50%, a civil war may begin.")
-            
+
+        elif command == "!quit":
+                print("Exiting.")
+                return
```

Comment:
Wouldn't this just print "Exiting." instead of actually quitting?

Reply:
Actually the whole game is inside a function so the return statement will exit the function and thus terminates the game

Reply:
Cheers, that makes sense now.